frontend/public/components/utils/resource-log.tsx

```diff
@@ -108,8 +145,69 @@ export const LogControls: React.FC<LogControlsProps> = ({
             <TogglePlay active={status === STREAM_ACTIVE} onClick={toggleStreaming} />
           )}
           {t(streamStatusMessages[status])}
-        </div>
+        </>
+      );
+    }
+    return (
+      <>
+        {t(streamStatusMessages[STREAM_EOF])}{' '}
+        {isFullscreen ? null : (
+          <Popover bodyContent={t('public~Play and pause are not available for historical logs.')}>
```

Comment:
Hey @megan-hall, I was wondering if we should consider removing this popover. The icon makes it look like a question ("Log stream ended?"), and I think "Log stream ended" on its own explains why there are no play or pause buttons. It's also what we show for current logs when the pod is complete. It's not specific to previous logs.

Reply:
Good point. Sounds good to me!

Reply:
@spadgett @megan-hall I removed the popover next to "Log stream ended".  Updated the PR description.